.horde.yml

```diff
@@ -49,10 +49,13 @@ dependencies:
       horde/xml_element: ^3
   optional:
     composer:
-      horde/activesync: ^3
       horde/http: ^3
```

Comment:
If possible, the optionals should also be in dev.
Mind: Composer's "optional" is actually supposed to contain a "reason", not a "version" as the value. But this would break pear xml. I am still figuring how to handle this.

Reply:
BTW: Why did you remove ActiveSync from the Optional block?

Reply:
that might have been by mistake. I added it back now